### PR TITLE
travis: prep for cross-compiles to succeed

### PR DESCRIPTION
**Pull request by vbatts**
This might start succeeding after #1655 

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -72,7 +72,6 @@ jobs:
   allow_failures:
     - os: osx
     - go: tip
-    - env: CROSS_PLATFORM=true
```
Comment (wking) [inline .travis.yml]: You should probably delete the `CROSS_PLATFORM` variables above as well; I don't think they do anything other than drive this switch.

Comment (vbatts) [reply .travis.yml]: done

**Comment by vbatts** [comment]
rebased

**Comment by umohnani8** [comment]
@vbatts can you please rebase again.

**Comment by vbatts** [comment]
oh my. sure thing.

**Comment by vbatts** [comment]
rebased

**Comment by vbatts** [comment]
build error looks unrelated https://travis-ci.org/kubernetes-sigs/cri-o/jobs/444060164#L625

**Comment by saschagrunert** [comment]
I would like to support here. It seems that some dependencies we added in the past (like the containerd stuff) do not build on macOS and Windows. What do we want to do about this? Is macOS and Windows really a thing we want to support in the future? If no, what else do we want to cross compile?

**Comment by rhatdan** [comment]
@vbatts should we close this?

**Comment by vbatts** [comment]
I suppose so

-------- Original Message --------

**Comment by umohnani8** [comment]
Closing then.

**Pull request closed without merging**